src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java

```diff
@@ -146,7 +146,7 @@ public void summaryReport_HappyPath() throws Exception {
                             .andExpect(model().attribute("sectionId", is(sectionId.toString())))
                             .andExpect(model().attribute("student",
                                             allOf(
-                                                    hasProperty("studentId",is(existingStudent.getStudentId())),
+                                                    hasProperty("studentId",is(studentId)),
                                                     hasProperty("attendances", hasSize(1)),
                                                     hasProperty("attendances",
                                                             containsInAnyOrder(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "sectionId" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)